File: web/src/render/codeSpanRenderer.ts
import type MarkdownNode from "../types/MarkdownNode";
import type Renderer from "../types/Renderer";
import type RendererState from "../types/RendererState";
import renderTag from "./renderTag";

const renderer: Renderer = {
	name: "code_span",
	render,
};
export default renderer;

function render(node: MarkdownNode, state: RendererState): void {
	renderTag(node, state, "code", false);
}

File: dotnet/Allmark/Render/CodeSpanRenderer.cs
namespace Allmark.Render;

using Allmark.Types;

public static class CodeSpanRenderer
{
    public static OutputRenderer Create()
    {
        return new OutputRenderer
        {
            Name = "code_span",
            Render = Render,
        };
    }

    public static void Render(MarkdownNode node, RendererState state, bool? decode = true)
    {
        RenderTag.Execute(node, state, "code", false);
    }
}
